File: java/FibonacciSeries.java
import java.util.Scanner;

public class FibonacciSeries {
    public static void fibonacci(int number){

        if(number <= 0){
            System.out.println("Enter a positive number.");
            return;
        }

        int num1 = 0;
        int num2 = 1;

        if(number >= 1){
            System.out.print(num1 + " ");
        }
        if(number >= 2){
            System.out.print(num2 + " ");
        }

        for(int i=2; i < number; i++){
            int sum = num1+num2;
            num1 = num2;
            num2 = sum;
            System.out.print(num2 + " ");
        }
    System.out.println();
    }    

    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int user_number = sc.nextInt();

        fibonacci(user_number);
        sc.close();
    }
}

File: C/fibonacci_series.c
#include<stdio.h>
void fibonacci(int number){
    if(number<=0){
        printf("Enter a positive number.\n");
        return;
    }

    int num1 = 0;
    int num2 = 1;

    if(number>=1){
        printf("%d ", num1);
    }
    if(number>=2){
        printf("%d ", num2);
    }

    for(int i=2; i<number; i++){
        int sum = num1 +num2;
        num1 = num2;
        num2 = sum;
        printf("%d ", num2);
    }
    printf("\n");
}
int main(){
    int user_number;
    printf("Enter a number: ");
    scanf("%d", &user_number);

    fibonacci(user_number);
    return 0;
}